README.md

```diff
@@ -330,6 +330,18 @@ interface {
 }
 ```
 
+#### [Interview Node](https://zwave-js.github.io/node-zwave-js/#/api/driver?id=interviewnode)
+
+[compatible with schema version: 18+]
+
+```ts
+interface {
+  messageId: string;
+  command: "driver.interview_node";
```

Comment:
Do we know why Z-Wave JS chose to put this command on the driver instead of the controller?

Reply:
I don't, but looking at the doc, this is the explanation for the driver:
The driver is the core of this library. It controls the serial interface, handles transmission and receipt of messages and manages the network cache. Any action you want to perform on the Z-Wave network must go through a driver instance or its associated nodes.

So I guess it makes sense that the driver is the one to do the interview

Reply:
But the inclusion command eg is on the controller?

Reply:
Good point... @AlCalzone ?

Reply:
I agree it is a bit inconsistent. I might move this to the controller or node class in v10.
The reason it is on the driver is that the method existed there before but was private. Now the private method was renamed and exposed in-place with an additional check.

Reply:
do we want to wait to introduce this until v10 in the server then? It might be better than creating a future breaking change for the server and lib

Reply:
I'm ok with waiting, but I don't have a strong opinion either way. Good to make it consistent and put on the controller. 👍

Reply:
We have no plans to use this anytime soon, we just generally try to stay at parity with zwave-js. @marcus-j-davies is this something you will need now or can you wait until v10?

Reply:
@raman325,

I don't plan on exposing the ability to throttle new node interviews in the .NET wrapper anytime soon - the `interview_node` command therefore, won't be missed at this time.

I will be releasing 3.1.0 and this has no changes with the offerings, so... all good my send 👍

Reply:
@AlCalzone is v10 coming soonish? If so, I'd say then we skip adding that command until it gets moved

Reply:
You can also consider adding it under the new name

Reply:
yeah but that feels like unnecessary tech debt. If someone comes with a compelling use case for the command today, it's easy enough to add, but if we know we are going to change it later, why bother?

Reply:
if you put it under the new name that AlCalzone is going to use, it won't have to be renamed again.

Reply:
I hope to finish v10 this week. Probably best to skip adding the command for now and add it with the bump to v10 IMO.